### ISSUE TITLE
plugin totally messed up

### ISSUE THREAD

**Issue by PhanaticD** [games647/ScoreboardStats#220]
I have been getting the concurrentmodification that everyone else has been getting, but the fix appears to be in a new version thats not at all ready, compiling the latest version off github is completely messed up, the plugin.yml generates wrong, after manually fixing that the sql.yml file doesnt get made, and using the old configs there are numerous sql related errors and the plugin doesnt function

basically, is there any way to get version 0.10.6 but with the concurrent exception fix?
